pyprophet/export_parquet.py

```diff
@@ -0,0 +1,255 @@
+    columns['FEATURE_MS1'] = ['APEX_INTENSITY', 'VAR_MASSDEV_SCORE', 'VAR_MI_SCORE', 'VAR_MI_CONTRAST_SCORE', 'VAR_MI_COMBINED_SCORE', 'VAR_ISOTOPE_CORRELATION_SCORE', 'VAR_ISOTOPE_OVERLAP_SCORE', 'VAR_IM_MS1_DELTA_SCORE', 'VAR_XCORR_COELUTION', 'VAR_XCORR_COELUTION_CONTRAST', 'VAR_XCORR_COELUTION_COMBINED', 'VAR_XCORR_SHAPE', 'VAR_XCORR_SHAPE_CONTRAST', 'VAR_XCORR_SHAPE_COMBINED']
+
+    # check if IM columns exist
+    query = con.execute("select count(*) as cntrec from pragma_table_info('feature_MS2') where name='EXP_IM'")
+    hasIm = query.fetchone()[0] > 0
+    if hasIm:
+        imColumns = ['EXP_IM', 'DELTA_IM']
+        columns['FEATURE_MS2'] = columns['FEATURE_MS2'] + imColumns
+        columns['FEATURE_MS1'] = columns['FEATURE_MS1'] + imColumns
+
+    ### pyprophet scores 
+    columns['SCORE_MS2'] = ["SCORE", "RANK", "PVALUE", "QVALUE", "PEP"]
+
+    # Check for Peptide/Protein scores Context Scores
+    if check_sqlite_table(con, "SCORE_PEPTIDE"):
+        pepTable = getPeptideProteinScoreTable(condb, "peptide")
+        pepJoin = 'LEFT JOIN pepTable ON pepTable.PEPTIDE_ID = PEPTIDE.ID and pepTable.RUN_ID = RUN.ID'
+        columns['pepTable'] = list(set(pepTable.columns).difference(set(['PEPTIDE_ID', 'RUN_ID']))) # all columns except PEPTIDE_ID and RUN_ID
+
+
+    if check_sqlite_table(con, "SCORE_PROTEIN"):
+        protTable = getPeptideProteinScoreTable(condb, "protein")
+        protJoin = 'LEFT JOIN protTable ON protTable.PROTEIN_ID = PROTEIN.ID and protTable.RUN_ID = RUN.ID'
+        columns['protTable'] = list(set(protTable.columns).difference(set(['PROTEIN_ID', 'RUN_ID']))) # all columns except PEPTIDE_ID and RUN_ID
+
+    ## other
+    columns['RUN'] = ['FILENAME']
+
+
+    ## mappings
+    columns['PRECURSOR_PEPTIDE_MAPPING'] = ['PEPTIDE_ID', 'PRECURSOR_ID']
+    columns['PEPTIDE_PROTEIN_MAPPING'] = ['PROTEIN_ID']
+    if check_sqlite_table(con, "GENE") and pd.read_sql('SELECT GENE_NAME FROM GENE', con).GENE_NAME[0]!='NA':
+        columns['PEPTIDE_GENE_MAPPING'] = ['GENE_ID']
+
+    # transition level
+    if transitionLevel:
+        columns['FEATURE_TRANSITION'] = ['AREA_INTENSITY', 'TOTAL_AREA_INTENSITY', 'APEX_INTENSITY', 'TOTAL_MI', 'VAR_INTENSITY_SCORE', 'VAR_INTENSITY_RATIO_SCORE', 'VAR_LOG_INTENSITY', 'VAR_XCORR_COELUTION', 'VAR_XCORR_SHAPE', 'VAR_LOG_SN_SCORE', 'VAR_MASSDEV_SCORE', 'VAR_MI_SCORE', 'VAR_MI_RATIO_SCORE', 'VAR_ISOTOPE_CORRELATION_SCORE', 'VAR_ISOTOPE_OVERLAP_SCORE']
```

Comment:
Same comment as above. I know it's currently not implemented for IPF, but I have added more IM scores to the `FEATURE_TRANSITION` table, so would be nice, not having to think about needing to manually keep track and add these scores.